src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/tests/latest/test_vm_commands.py

```diff
@@ -971,6 +971,43 @@ def test_vm_create_ubuntu(self, resource_group, resource_group_location):
         self.cmd('vm create --resource-group {rg} --admin-username {username} --name {vm} --authentication-type {auth} --image {image} --ssh-key-value \'{ssh_key}\' --location {loc} --data-disk-sizes-gb 1 --data-disk-caching ReadOnly  ')
 
 
+class VMCreateEphemeralOsDisk(ScenarioTest):
+
+    @ResourceGroupPreparer(name_prefix='cli_test_vm_create_ephemeral_os_disk')
+    def test_vm_create_ephemeral_os_disk(self, resource_group, resource_group_location):
+
+        self.kwargs.update({
+            'vm': 'cli-test-vm-local-1',
+            'vm_2': 'cli-test-vm-local-2',
+            'image': 'UbuntuLTS',
+            'ssh_key': TEST_SSH_KEY_PUB,
+            'loc': resource_group_location
+        })
+
+        # check that we can create a vm with local / ephemeral os disk.
+        self.cmd('vm create --resource-group {rg} --name {vm} --image {image} --ssh-key-value \'{ssh_key}\' --location {loc} --ephemeral-os-disk')
+
+        self.cmd('vm show -g {rg} -n {vm}', checks=[
+            self.check('provisioningState', 'Succeeded'),
+            self.check('osProfile.computerName', '{vm}'),
+            self.check('osProfile.linuxConfiguration.disablePasswordAuthentication', True),
+            self.check('osProfile.linuxConfiguration.ssh.publicKeys[0].keyData', '{ssh_key}'),
+            self.check('storageProfile.osDisk.caching', 'ReadOnly'),
```

Comment:
I would suggest remove those unrelated checks and just focus on the new settings